permissions/plugin-linchpin.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "linchpin"
+github: "jenkinsci/linchpin-plugin"
+paths:
+- "org/jenkins-ci/plugins/linchpin-plugin"
```

Comment:
Mismatch between this line and artifactId. Please see this repo's readme.

Reply:
Hi, "linchpin" is my artifactId.